plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/GitHookTasksTest.kt

```diff
@@ -157,8 +157,10 @@ class GitHookTasksTest : AbstractPluginTest() {
         build(":$INSTALL_GIT_HOOK_FORMAT_TASK").run {
             assertThat(task(":$INSTALL_GIT_HOOK_FORMAT_TASK")?.outcome).isEqualTo(TaskOutcome.SUCCESS)
             val hookText = gitDir.preCommitGitHook().readText()
-            assertThat(hookText).contains("git stash push")
-            assertThat(hookText).contains("git stash pop")
+            assertThat(hookText).contains("git diff > unstaged.diff")
```

Comment:
I am really need to come up with git hook test setup that actually runs installed githook on some test project :thinking:

Reply:
#503